src/rnp/rnpcfg.cpp

```diff
@@ -284,12 +288,26 @@ rnp_cfg_addstr(rnp_cfg_t *cfg, const char *key, const char *str)
 }
 
 const char *
-rnp_cfg_getstr(const rnp_cfg_t *cfg, const char *key)
+rnp_cfg_getstr(const rnp_cfg_t *cfg, const char *key, const size_t item_idx)
 {
     rnp_cfg_item_t *it = rnp_cfg_find(cfg, key);
 
     if (it && (it->val.type == RNP_CFG_VAL_STRING)) {
-        return it->val.val._string;
+        if (item_idx == 0) {
```

Comment:
The same as previous comment - you don't need to modify this function since all required functionality is already implemented.